### PR TITLE
Add example of using a custom token on the standard bridge

### PR DESCRIPTION
**Pull request by elenadimitrova**
This example is about demonstrating how to customise the `L2StandardToken` implementation and deploy it.

Closes OP-820

### CODE REVIEW

File: standard-bridge-custom-token/README.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+```
+import { L2StandardERC20 } from "@eth-optimism/contracts/libraries/standards/L2StandardERC20.sol";
+```
+
+Then the only thing we need to do is call the internal `_setupDecimals(8)` method to alter the token `decimals` property from the default `18` to `8`.
+
+## Deploying the Custom Token
+
+Deployment script is made available under `scripts/deploy-custom-token.js` that you can use to instantiate `L2CustomERC20` both on a local dev node or on `optimism-kovan`.
```
Comment (qbzzt) [inline standard-bridge-custom-token/README.md:26]: `both` takes `and`. For `or` you should use `either`.

File: standard-bridge-custom-token/README.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+You should already have a Hardhat development environment, as explained in
+[the tutorial](https://github.com/ethereum-optimism/optimism-tutorial/tree/main/hardhat).
+
+### The Configuration File
+
+The hardhat config here `hardhat.config.js` is already setup to run against local dev environment, `optimism-kovan` and `optimism-mainnet` networks.
+
+### The .env File
+
+Create an .env file in the root of `standard-bridge-custom-token` folder and add `PRIVATE_KEY` to it. This account is going to be used to call the factory and create your L2 ERC20.
```
Comment (qbzzt) [inline standard-bridge-custom-token/README.md:42]: To use a network (either Optimistic Kovan or Optimistic Ethereum), create an ... 

Maybe after this paragraph remind them to fund that account?

File: standard-bridge-custom-token/README.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+Run the following script
+
+```sh
+yarn hardhat run scripts/deploy-custom-token.js --network optimism-kovan
+```
+
+At the end you should get a successful output confirming your token was created and the L2 address:
+
+`L2CustomERC20 deployed to: 0x5CFE8703A62E3a80ab7233263C074698b722d48b`
+
```
Comment (qbzzt) [inline standard-bridge-custom-token/README.md:62]: Maybe a link to the ERC20 send and receive tutorial so they'll know the next step is to verify that ERC20 tokens flow between the two?

File: standard-bridge-custom-token/README.md (hunk 2 above)
Comment (elenadimitrova) [reply standard-bridge-custom-token/README.md:42]: Updated the same readme section for `standard -bridge-standard-token` to be consistent.

File: standard-bridge-custom-token/README.md (hunk 3 above)
Comment (elenadimitrova) [reply standard-bridge-custom-token/README.md:62]: Good idea

**Review by qbzzt — APPROVED** [review]
Just rename the networks to optimisic, optimistic-kovan, etc. (we're trying to get people to distinguish Optimisic Ethereum the protocol from Optimism PBC the company) and it's ready to merge.

**Pull request merged**